src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -20,6 +20,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="AllocationHelper.cs" />
+    <Compile Include="BuffersExtensions\BuffersExtensions.cs" />
```

Comment:
I would rename this file to BuffersExtensionsTests.cs

Reply:
Also, can you move this to its own itemgroup, just like MemoryPool? https://github.com/dotnet/corefx/blob/5e7b002d6b5ddbb41a47be7f5f35b05c0437551a/src/System.Memory/tests/System.Memory.Tests.csproj#L199

While you are moving them, the ReadOnlyBuffer test files should also be in its own itemgroup with the rest of them. Feel free to fix that too :)
https://github.com/dotnet/corefx/blob/5e7b002d6b5ddbb41a47be7f5f35b05c0437551a/src/System.Memory/tests/System.Memory.Tests.csproj#L88-L97